### ISSUE TITLE
Support for accessors

### ISSUE THREAD

**Issue by ekvedaras** [laravel-idea/plugin#353]
It would be really awesome if the plugin could resolve references for [accessors](https://laravel.com/docs/8.x/eloquent-mutators#defining-an-accessor) / [mutators](https://laravel.com/docs/8.x/eloquent-mutators#defining-a-mutator).

So it would include them in autocomplete and jump to method definition on click.

I think this is a Laravel feature that is used quite often so would be beneficial for many 🙂 

I couldn't find any issues regarding this on this repo yet 🤔

**Comment by smares** [comment]
I hope @adelf is not going to stone me, but maybe accessors can also be implemented for Livewire. The way it works is you use `$this->isBookmarked` in the Blade template for example, and you define a `public function getIsBookrmarkedProperty()` in the Livewire component.

**Comment by adelf** [comment]
@ekvedaras, I think accessors already working... Eloquent helper code generator should understand them. I’ll check.

@smares, it’s definitely a different task)

**Comment by ekvedaras** [comment]
OK, I tried to generate (because I suspected that might be the case) but it's actually crashing for me. Let me try again, I'll post the exception if it's still there.

**Comment by ekvedaras** [comment]
Here it is:

<details>
<summary><code>java.util.NoSuchElementException: List is empty.</code></summary>

```
java.util.NoSuchElementException: List is empty.
	at kotlin.collections.CollectionsKt___CollectionsKt.first(_Collections.kt:212)
	at com.laravel_idea.plugin.helperCodeGeneration.d.b$a.a(b$a.java:57)
	at com.laravel_idea.plugin.helperCodeGeneration.d.b$a.invoke(b$a.java:11)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at com.laravel_idea.plugin.helperCodeGeneration.d.b.m(b.java)
	at com.laravel_idea.plugin.helperCodeGeneration.d.d.a(d.java:316)
	at com.laravel_idea.plugin.helperCodeGeneration.d.d.a(d.java:75)
	at com.laravel_idea.plugin.helperCodeGeneration.d.f.a(f.java:76)
	at com.laravel_idea.plugin.generation.models.settings.specialGenerations.models.IdeHelperModelsGeneration$c$a.a(IdeHelperModelsGeneration$c$a.java:94)
	at com.laravel_idea.plugin.generation.models.settings.specialGenerations.models.IdeHelperModelsGeneration$c$a.compute(IdeHelperModelsGeneration$c$a.java:85)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:841)
	at com.laravel_idea.plugin.generation.models.settings.specialGenerations.models.IdeHelperModelsGeneration$c.a(IdeHelperModelsGeneration$c.java:93)
	at com.laravel_idea.plugin.generation.models.settings.specialGenerations.models.IdeHelperModelsGeneration$c.compute(IdeHelperModelsGeneration$c.java:85)
	at com.intellij.openapi.progress.Task$WithResult.run(Task.java:349)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:450)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:117)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressSynchronously$8(CoreProgressManager.java:556)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$new$0(ProgressRunner.java:81)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:243)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:183)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:705)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:647)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:63)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:170)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:243)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
```
</details>

Am I missing something? 🤔

**Comment by adelf** [comment]
> Am I missing something?

Plugin version:)

**Comment by ekvedaras** [comment]
Hehe 🙂 Latest I think `4.3.4.212`

**Comment by adelf** [comment]
Fixed. Thank you. I'll release a new version in the next week.

**Comment by adelf** [comment]
![image](https://user-images.githubusercontent.com/2818394/128636120-5da5aaa6-e927-4c09-ab73-5372f835dfd1.png)

`active()` also goes to `scopeActive()` method. Thank you for pushing this :)

**Comment by adelf** [comment]
Implemented in the 4.4 version.
Full list of changes - https://laravel-idea.com/blog/2021/4-4-release
